pip/templates/deploy.py

```diff
@@ -76,8 +82,8 @@ def upload_command(repo_type_key, package_file, wheel_file):
 with open("{version_file}") as version_file:
     version = version_file.read().strip()
 try:
-    new_package_file = dist_dir + "/{package_file}".replace(".tar.gz", "-{}.tar.gz".format(version))
-    new_wheel_pep491 = dist_dir + "/{wheel_file}".replace("-", "_").replace(".whl", "-{}-py3-none-any.whl".format(version))
+    new_package_file = dist_dir + "/{package_file}".replace("{suffix}", "").replace(".tar.gz", "-{}.tar.gz".format(version))
+    new_wheel_pep491 = dist_dir + "/{wheel_file}".replace("{suffix}", "").replace("-", "_").replace(".whl", "-{}-{distribution_tag}.whl".format(version))
```

Comment:
Let's rename these to `source_package` and `wheel_package`


Let's rename `package_file` to `source_package_name` and `wheel_file` to `wheel_package_name`